client/src/pages/Profile.js

```diff
@@ -1,147 +1,170 @@
-import ProjectList from './Project'
-
-import LocationOnIcon from '@material-ui/icons/LocationOn';
-import JHAvatar from '../assets/jh-avatar.jpg'
-import coffeeCup from '../assets/coffee-cup.jpg'
-import espresso from '../assets/espresso.jpg'
-import espresso2 from '../assets/espresso2.jpg'
-import pouringCoffee from '../assets/pouring-coffee.jpg'
-import "./Profile.css"
-
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import Avatar from "@material-ui/core/Avatar";
+import Chip from "@material-ui/core/Chip";
+import Container from "@material-ui/core/Container";
+import { Typography } from "@material-ui/core";
+import Button from "@material-ui/core/Button";
+
+import Fields from "./Fields";
+import ProjectList from "./Project";
+
+import LocationOnIcon from "@material-ui/icons/LocationOn";
+import JHAvatar from "../assets/jh-avatar.jpg";
+import coffeeCup from "../assets/coffee-cup.jpg";
+import espresso from "../assets/espresso.jpg";
+import espresso2 from "../assets/espresso2.jpg";
+import pouringCoffee from "../assets/pouring-coffee.jpg";
+import "./Profile.css";
 
 const useStyles = makeStyles({
   avatar: {
-    margin: 10,
+    margin: 10
   },
   bigAvatar: {
     margin: 10,
     width: 60,
-    height: 60,
+    height: 60
   },
 
   projectTitle: {
-    padding: 20,
+    padding: 20
   },
   projectGridList: {
-    padding:20,
+    padding: 20
   }
 });
 
-const fieldsData = ["TECHNOLOGY", "COFFEE", "CUSTOMER SERVICE", "RESTAURANT"]
+const fieldsData = ["TECHNOLOGY", "COFFEE", "CUSTOMER SERVICE", "RESTAURANT"];
 
 const projectData = [
   {
     img: coffeeCup,
-    alt: 'Coffee Cup',
-    field: 'Customer Service',
+    alt: "Coffee Cup",
+    field: "Customer Service",
     title: "Urban Jungle: eco-friendly coffee shop",
-    raised: '23,874',
-    goal: '40,000',
-    equity: '10%',
-    daysToGo: '44 days to go'
+    raised: "23,874",
+    goal: "40,000",
+    equity: "10%",
+    daysToGo: "44 days to go"
   },
   {
     img: espresso,
-    alt: 'Espresso',
-    field: 'Coffee',
-    title: 'Energy Run: the quickest coffee experience',
-    raised: '7,257',
-    goal: '12,383',
-    equity: '13%',
-    daysToGo: '19'
+    alt: "Espresso",
+    field: "Coffee",
+    title: "Energy Run: the quickest coffee experience",
+    raised: "7,257",
+    goal: "12,383",
+    equity: "13%",
+    daysToGo: "19"
   },
   {
     img: pouringCoffee,
-    alt:'Pouring Coffee',
-    field: 'Restaurant',
-    title: 'Energy Rush: an even quicker coffee experience',
-    raised: '34,912',
-    goal: '50,000',
-    equity: '5%',
-    daysToGo: '5'
+    alt: "Pouring Coffee",
+    field: "Restaurant",
+    title: "Energy Rush: an even quicker coffee experience",
+    raised: "34,912",
+    goal: "50,000",
+    equity: "5%",
+    daysToGo: "5"
   }
-] 
+];
+
+// Purposely didn't put the following as a part of a function's body
```

Comment:
You would put that inside a `useEffect` hook inside  profilePage, so that it runs when the component finishes loading (https://reactjs.org/docs/hooks-effect.html, https://medium.com/@felippenardi/how-to-do-componentdidmount-with-react-hooks-553ba39d1571). Then, once you have the response, you save in the state using a `useState` hook (https://reactjs.org/docs/hooks-state.html)